Sources/BothamPullToRefreshHandler.swift

```diff
@@ -9,15 +9,18 @@
 import Foundation
 
 public class BothamPullToRefreshHandler: NSObject {
-    let presenter: BothamPullToRefreshPresenter
-    let refreshControl = UIRefreshControl()
+    private let presenter: BothamPullToRefreshPresenter
+    private let refreshControl = UIRefreshControl()
+    private var scrollView: UIScrollView?
+    
 
     public init(presenter: BothamPullToRefreshPresenter) {
         self.presenter = presenter
         super.init()
     }
 
     public func addTo(scrollView: UIScrollView) {
```

Comment:
@pedrovgs should we rename this method, to something like setTo(...), because the handler can be used by only one scrollview at the time ?